tests/unit/controllers/api/v1/test_alias_execution_rbac.py

```diff
@@ -36,7 +37,7 @@
 
 EXECUTION = ActionExecutionDB(id='54e657d60640fd16887d6855',
                               status=LIVEACTION_STATUS_SUCCEEDED,
-                              result='')
```

Comment:
Thanks for fixing that.

And yeah I was somewhat shocked when I saw how many cases like that we had in st2 core tests and mongoengine just ignoring invalid type even though it should have only accepted dicts.